### ISSUE TITLE
sdxl webui cannot generate GIF or MP4

### ISSUE THREAD

**Issue by ustcxiexk** [guoyww/AnimateDiff#223]
Error Info：

```
……stable-diffusion-webui/extensions/sd-webui-animatediff-master/scripts/animatediff.py", line 71, in postprocess_batch_list
        self.prompt_scheduler.save_infotext_img(p)
    AttributeError: 'NoneType' object has no attribute 'save_infotext_img'

……extensions/sd-webui-animatediff-master/scripts/animatediff.py", line 83, in postprocess
        self.prompt_scheduler.save_infotext_txt(res)
    AttributeError: 'NoneType' object has no attribute 'save_infotext_txt'
```

webui output：only image，no gif or mp4 video.

**Comment by Seryogerr** [comment]
Same error in runpod
And generate only random img like "Batch count"
Holly, I just realized I hadn't updated the version :D

**Comment by ustcxiexk** [comment]
> Same error in runpod And generate only random img like "Batch count" Holly, I just realized I hadn't updated the version :D

Does it work? I have updated sd-webui-animatediff, the images are still random, cannot generate a video.

**Comment by ythtisotone** [comment]
> > Same error in runpod And generate only random img like "Batch count" Holly, I just realized I hadn't updated the version :D
> 
> Does it work? I have updated sd-webui-animatediff, the images are still random, cannot generate a video.

same with you,   and may be the ckpt problem.


Exception: bad file inside D:\SD\sd-webui\extensions\sd-webui-animatediff\model\mm_sdxl_v10_beta.ckpt: mm_sdxl_nightly,ckpt/byteorder

AttributeError: 'NoneType' object has no attribute 'pop'

then the same error info above.

**Comment by MelleNi** [comment]
I have the same issue with webui, it just generates a bunch of separate images, not sequential in any way

**Comment by TijuanaKez** [comment]
Exact same issue. 

`AttributeError: 'NoneType' object has no attribute 'pop'`

Then just does a batch of images with different seeds.

**Comment by droide** [comment]
I solved changed model, because i think not all model are compatible with Animatediff

**Comment by Whiterain1000** [comment]
Same error with the mm_sdxl_v10_beta.ckpy and the animatediffMotion_sdxlV10Beta.ckpt from civit. Just a batch run of images no output.

**Comment by dongkuang** [comment]
why?

**Comment by JoshonSmith** [comment]
> I solved changed model, because i think not all model are compatible with Animatediff

How to do?please help

**Comment by TijuanaKez** [comment]
> I solved changed model, because i think not all model are compatible with Animatediff

Well animatediff supposedly works with SDXL now, hence the animatediffMotion_sdxlV10Beta.ckpt model. I'm just using with the vanilla SDXL base model from hugging face and it's throwing that error every time.

**Comment by Sbbke** [comment]
In my case, the problem was caused by the motion model was considered unsafe when verifying the file, so the program is not going to read it, just skip the check by simply adding "--disable-safe-unpickle" into commandline argument, then the problem is solved.  ( By the way, I've read an article online saying that this problem can be solved by upgrading the torch and xformers, it didn't work for me though.)

**Comment by TijuanaKez** [comment]
> In my case, the problem was caused by the motion model was considered unsafe when verifying the file, so the program is not going to read it, just skip the check by simply adding "--disable-safe-unpickle" into commandline argument, then the problem is solved. ( By the way, I've read an article online saying that this problem can be solved by upgrading the torch and xformers, it didn't work for me though.)

yes I'd mentioned that above re "--disable-safe-unpickle", but turning this off allows execution of malicious code. It's like saying, just turn off your anti-virus software = problem solved. Not really.

**Comment by Sbbke** [comment]
> > In my case, the problem was caused by the motion model was considered unsafe when verifying the file, so the program is not going to read it, just skip the check by simply adding "--disable-safe-unpickle" into commandline argument, then the problem is solved. ( By the way, I've read an article online saying that this problem can be solved by upgrading the torch and xformers, it didn't work for me though.)
> 
> yes I'd mentioned that above re "--disable-safe-unpickle", but turning this off allows execution of malicious code. It's like saying, just turn off your anti-virus software = problem solved. Not really.

Sorry, somehow I missed that message. I agree that it is not a solution to add in "--disable-safe-unpickle", is just a temporary measure for animetediff to work, after I disabled the safe check, a lot of problems occurred, like the webui required a total restart after animatediff for a normal t2i to work.

**Comment by ustcxiexk** [comment]
This solves the problem: https://github.com/continue-revolution/sd-webui-animatediff/issues/379
